src/main/java/core/mate/academy/service/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class BulldozerProducerImpl implements MachineProducer<Bulldozer> {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line above the class declaration.